Cloudflare/Fetch-Canary-Syslog.ps1

```diff
@@ -16,7 +16,7 @@ $Response = Invoke-WebRequest -Uri $WorkerURL -Headers @{ 'auth' = $WorkerAuth}
 if($Response.Statuscode -eq 204){exit 0} # Nothing to process - resultset is empty
  
 $Content = $Response.Content
-$SyslogArray = ($Content -split "\r?\n|\r")
+$SyslogArray = ($Content -split '\r?\n')
```

Comment:
Could you explain why this is necessary here? Perhaps with an example of what you're fixing / changing? Thanks

Reply:
Hi Max, for instances where there is only a single returned value, this was causing a blank value.  This resolved the issue for my test, but I may end up reworking the code in the long run.

Reply:
This was rewritten